### ISSUE TITLE
[api-extractor] @inheritdoc does not work as documented

### ISSUE THREAD

**Issue by acchou** [Microsoft/web-build-tools#1174]
The [example in the documentation](https://api-extractor.com/pages/tsdoc/syntax/#inheritdoc) for `{@inheritdoc}` doesn't appear to work. The documentation for the interface shows up, but the class methods remain undocumented in the output despite having `{@inheritdoc IWidget}`.

Inheritdoc also seems to be broken for the generated documentation for `web-build-tools`. For example the API docs for [`IExtractorConfig.policies`](https://microsoft.github.io/web-build-tools/api/api-extractor.iextractorconfig.policies.html) has no documentation, but in the source code it's supposed to inherit documentation from [`IExtractorPoliciesConfig`](https://microsoft.github.io/web-build-tools/api/api-extractor.iextractorpoliciesconfig.html):

```
  // Excerpt from web-build-tools/apps/api-extractor/src/api/IExtractorConfig.ts:
  /**
   * {@inheritDoc IExtractorPoliciesConfig}
   */
  policies?: IExtractorPoliciesConfig;
```

I'm using these versions:
```
        "@microsoft/api-documenter": "^7.0.35",
        "@microsoft/api-extractor": "^7.0.28",
```

**Comment by octogonz** [comment]
API Extractor 7 is still tagged as `beta`.  After PR 1173, there's two more PRs I need to do before it will be "code complete".  Support for `@inheritDoc` is one of them.  :-)

For the initial release, we may only support `@inheritDoc` within a package: API Extractor 6 allows you to also inherit from declarations that are imported from another package, which is pretty useful.  However, it requires you to publish .api.json files with those packages.  The .api.json files can be somewhat large and you don't want your "internal" docs in there, but then `@inheritDoc` won't work with internal members.  For API Extractor 7.1, I wanted to instead explore the idea of parsing the inherited content directly from `/** */` comments, which would eliminate the reliance on .api.json.  But this requires running the compiler against source files that are outside the active context, which needs a little thought.  (That's why I punted it from 7.0.)

**Comment by acchou** [comment]
Thanks, I look forward to `@inheritDoc` coming to AE7. 

Supporting `@inheritDoc` to imported packages sounds like a great feature, but for my own use case it's not something I need. Often if you need to link to an external package, it's possible to provide a regular `@link` to the public documentation for the package. But I could see how it could be very convenient for monorepos using rush or lerna.

**Comment by octogonz** [comment]
Here's a PR: https://github.com/Microsoft/web-build-tools/pull/1181

**Comment by octogonz** [comment]
@acchou This was published with API Extractor beta 7.0.33.  Give it a try and lemme know how it works.
